src/main/java/com/hamarb123/macos_input_fixes/ModOptions.java

```diff
@@ -504,6 +504,20 @@ private static void loadInterface()
 
 	//saving and loading of options:
 
+	private static String getStringSafely(String key, NbtCompound compoundTag)
+	{
+		if (FabricReflectionHelper.Has_NbtCompound_getStringOr())
+		{
+			//1.21.5+
+			return FabricReflectionHelper.NbtCompound_getStringOr(key, "");
+		}
+		else
+		{
+			//1.14-1.21.4
+			return compoundTag.getString(key);
```

Comment:
This should be done via reflection also, so we don't have a static reference to a method that might not exist.